cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -91,4 +91,19 @@ package object cwl extends TypeAliases {
     import wdl4s.cwl.Implicits.enumerationEncoder
     commandLineTool.asJson
   }
+
```

Comment:
@danbills For me it is mainly that I've seen too many package objects become giant bags of random stuff. Once that point hits it's impossible to say "don't put that here" because everything else is already in there.